libcudacxx/.upstream-tests/test/std/skip_nvrtc_launch.pass.cpp

```diff
@@ -0,0 +1,32 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Part of libcu++, the C++ Standard Library for your entire system,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+//
+// UNSUPPORTED: libcpp-has-no-threads
+// NVRTC_SKIP_KERNEL_RUN // do compile, but do not run under nvrtc
```

Comment:
I believe we should add:
```suggestion
// UNSUPPORTED: !nvrtc
// NVRTC_SKIP_KERNEL_RUN // do compile, but do not run under nvrtc
```

Then we can simply remove the #ifdef in line 25